safe/impact_function/impact_function.py

```diff
@@ -2356,3 +2466,118 @@ def append_ISO19115_keywords(keywords):
         for key, value in ISO19115_mapping.items():
             ISO19115_keywords[value] = setting(key, expected_type=str)
         keywords.update(ISO19115_keywords)
+
+    @staticmethod
+    def load_from_output_metadata(output_metadata):
+        """Set Impact Function based on an output of an analysis's metadata.
+
+        :param output_metadata: Metadata from an output layer.
+        :type output_metadata: OutputLayerMetadata
+
+        :returns: Impact Function based on the metadata.
+        :rtype: ImpactFunction
+        """
+        impact_function = ImpactFunction()
+        provenance = output_metadata['provenance_data']
+        # Set exposure layer
+        exposure_path = get_provenance(provenance, provenance_exposure_layer)
+        if exposure_path:
```

Comment:
Do we need these 'IF' ? AFAIK, this path will be always defined, or we can't create the IF and we should stop the process. No?

Reply:
I am not sure either actually.
If the user clean or move the original layers of input layers, we can skip this. Actually, we only need the output's layer to generate the report.
But, honestly, I am not sure about what's the best behaviour.